nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/SplitText.java

```diff
@@ -134,6 +168,28 @@ protected void init(final ProcessorInitializationContext context) {
     }
 
     @Override
+    protected Collection<ValidationResult> customValidate(ValidationContext validationContext) {
+        ArrayList<ValidationResult> results = new ArrayList<>();
+
+        results.add(new ValidationResult.Builder()
+            .subject("Remove Trailing Newlines")
+            .valid(!validationContext.getProperty(REMOVE_TRAILING_NEWLINES).asBoolean())
+            .explanation("Property is deprecated; value must be set to false. Future releases may remove this Property.")
+            .build());
+
+        final boolean invalidState = (validationContext.getProperty(LINE_SPLIT_COUNT).asInteger() == 0
+                && validationContext.getProperty(FRAGMENT_MAX_SIZE).asDataSize(DataUnit.B) == null);
```

Comment:
I think this line should be:
&& !validationContext.getProperty(FRAGMENT_MAX_SIZE).isSet() );